.gitlab/custom-jobs-and-variables.yml

```
@@ -49,7 +49,7 @@ variables:
 # Arguments for job level allocation
   TIOGA_JOB_ALLOC: "--nodes=1 --begin-time=+5s"
 # Project specific variants for corona
-  PROJECT_TIOGA_VARIANTS: "~shared ~openmp +tests"
+  PROJECT_TIOGA_VARIANTS: "~shared ~openmp +vectorization +tests"
```

```suggestion
  PROJECT_TIOGA_VARIANTS: "~shared +openmp +vectorization +tests"
```
I think we want OpenMP turned on here

---

As you wish. This is a change compared to `develop`. Don’t you want openmp on corona too?